tsconfig.json

```
@@ -0,0 +1,14 @@
+{
+	"compilerOptions": {
+		"allowJs": true,
+		"checkJs": true,
+		"noEmit": true,
+		"strict": true,
+		"allowImportingTsExtensions": true,
+		"lib": ["dom", "es2023"],
+		"module": "nodenext"
+	},
+	"exclude": [
+			"./test/**",
+	]
```

YMMV:

```suggestion
	"exclude": ["./test/**"]
```

One might argue that if you have types, TDD should cover those as well - in the sense that it helps refine interfaces. I'm very much on the fence there, just because types are such a weird complexification. In this case, I believe excluding tests is probably the right choice.

---

If those were unit tests, I would agree. But in the CLI tests, I don't see a lot of value in type checks. Especially because they fail as TS doesn't know how to find the types.

---

Good point. 👌